src/core/config.js

```diff
@@ -66,6 +66,39 @@ export default function (vm) {
       : window.$docsify
   );
 
+  // Merge default and user-specified key bindings
+  if (config.keyBindings !== false) {
+    config.keyBindings = Object.assign(
+      // Default
+      {
+        focusContent: {
+          bindings: "'",
+          callback(e) {
+            const containerElm = document.querySelector('.markdown-section');
+            const focusElm = containerElm.querySelector(
+              'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"])'
+            );
```

Comment:
Personally, I have a little confused about the `'` (Focus on content area) behavior.
It more like ` toTop` for me since it will scroll to the content top.
And the `\\` , which hide the sidebar more like the `Focus on content area` to me.

Reply:
Fair point. I have removed the "Focus on content area" keybinding (<kbd>'</kbd>).

The original intention behind this keybinding was to allow visitors navigating with the keyboard to quickly move from different areas of the screen back to the content area. However, I've learned that screen readers offer their own methods of navigating between "landmark" elements such as the `<main>` (or in our case, `role="main"`) container, which arguably make a "focus on content area" shortcut unnecessary.